app/views/layouts/body/_contents.html.erb

```diff
@@ -1,3 +1,29 @@
 <div class="container" id="content-wrapper">
+  <% if flash %>
```

Comment:
What are your thoughts on moving all of these to either their own partial / helper, or making it so that a single partial is called multiple times with variables to change it up?  The code for each of the flash notification types looks so similar, it seems like it could be pulled out a bit.  Either way, might be nice to have it in a partial just so that the `_content` isn't full of alert code.